src/data/__fixtures__/movie.ts

```diff
@@ -0,0 +1,17 @@
+import { MovieData } from "../types";
+
+// interface MovieMaybe {
+//   name?: string;
+// }
+
+// let myMovie: MovieMaybe = {}
+// let myMovie2: MovieMaybe = { name: 'asldf'}
+
+export const createStubMovie = (overrides: Partial<MovieData>): MovieData => ({
```

Comment:
`Partial` is a generic type that:

1. Takes in an original type `T`
2. Creates a new type where:
3. For each `P` in `keyof T`:
4. The value there is `?` (optionally) `T[P]`